### PR TITLE
[IMP] web: list view: select all records on mobile

### PR DESCRIPTION
**Pull request by adr-odoo**
After this commit, as soon as at least one record is selected, the "All" button is available to select all records of the domain.

Note that it will not only select the records of the current page but all records of all pages.

Task-ID: 4314183

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by adr-odoo — COMMENTED** [review]
Missing test.

File: addons/web/static/src/core/pager/pager_indicator.scss [hunk 1]
```diff
@@ -1,6 +1,16 @@
 .o_pager_indicator {
     z-index: $zindex-modal + 1;
     transition: opacity 0.4s;
+
+    > .o_pager_indicator_inner {
+        min-width: 13ch;
+        padding: $o-touch-btn-padding;
+        text-align: center;
```
Comment (Iucapad) [inline addons/web/static/src/core/pager/pager_indicator.scss:8]: Can we use bootstrap classes when possible?
- Do we absolutely need to use a $o-touch-btn-padding
- Same for the bg color, radius and border

**Comment by Iucapad** [comment]
Thanks! @robodoo r+

**Comment by adr-odoo** [comment]
@robodoo r+

**Comment by adr-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**